src/System.Drawing.Common/tests/SystemFontsTests.cs

```diff
@@ -3,6 +3,7 @@
 
 using System.Collections.Generic;
 using System.Runtime.InteropServices;
+using Microsoft.VisualStudio.TestPlatform.CommunicationUtilities;
```

Comment:
Why do we need this using? It seems like you accidentally added it and is causing the build to fail.

Reply:
Oops. Will remove.